packages/core/src/providers/network/connectors/context.tsx

```diff
@@ -32,6 +69,13 @@ export const ConnectorContext = createContext<ConnectorContextValue>({
   //eslint-disable-next-line @typescript-eslint/no-empty-function
   reportError: () => {},
   isLoading: false,
+  setError: () => {
+    throw new Error('Function not implemented.')
+  },
+  active: false,
+  switchNetwork: () => {
+    throw new Error('Function not implemented.')
+  },
```

Comment:
should use usecallback
@yivlad

Reply:
It should not, this value that the context is initiallized with